assets/styles/style.css

```diff
@@ -1,36 +1,69 @@
 .topic h1 {
-	font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
-	font-size: 30px;
-	margin-bottom: 20px;
+  font-family: "Times New Roman", Times, serif;
+  font-size: 30px;
+  margin-bottom: 20px;
 }
 
 #background {
-	background-image: image("coder.jpg");
-	background-size: cover;
-	background-position: center;
+  background: linear-gradient(#7777, #7890), url("../images/coder.png");
+  background-size: cover;
+  background-position: center;
+  color: white;
 }
 
 .topic {
-	text-align: center;
+  text-align: center;
 }
+
 .topic p {
-	font-style: italic;
-	font-size: 20px;
-	margin-top: 10px;
+  font-style: italic;
+  font-size: 20px;
+  margin-top: 10px;
 }
 
 #informations {
-	padding-top: 10;
-	margin: 50px;
-	background-color: rgba(0, 0, 0, 50%);
-	
+  padding-top: 25px;
+  padding-left: 30px;
+  margin-right: 350px;
+  margin-left: 350px;
+  padding-right: 200px;
+  border-radius: 10px;
+  background-color: #011436;
+}
```

Comment:
Review your fonts (family, size, and weight), your line height in between the input fields, radio, and checkboxes. Also, modify the color of your submit button.

Reply:
ok

Reply:
Hello, bro your work looks better now but you still did not change the font family, weight, and size. Go on google fonts it will help you to give specific font families to your projects